REST/PowerShell/Variables/AddOrEditVariablesWithScoping.ps1

```diff
@@ -0,0 +1,194 @@
+Function Get-OctopusProject {
+    # Define parameters
+    param(
+        $OctopusServerUrl,
+        $ApiKey,
+        $ProjectName,
+        $SpaceId
+    )
+    # Call API to get all projects, then filter on name
+    $octopusProject = Invoke-RestMethod -Method "get" -Uri "$OctopusServerUrl/api/projects/all" -Headers @{"X-Octopus-ApiKey" = "$ApiKey" }
```

Comment:
You could make this slightly more efficient by changing the url to `$OctopusServerUrl/api/$spaceId/projects/all` and then you dont need to filter the space at the end?